docs/how-to/deploy/cloud-hosts/gcp-cloud-run.mdx

```diff
@@ -57,3 +57,12 @@ You can see the logs of the deployment in the "Logs" tab, and if the deployment
 
 We shared the YAML file that we used to deploy the PDP on GCP Cloud Run.
 Take a look at the following [Deployments Github Repository](https://github.com/permitio/permit-pdp-deployments-examples/blob/main/gcp/cloud-run.yaml)
+
+## Signal 6 in Cloud Run
+In some cases we encountered a behavior where Cloud Run unexpectedly interrupt the PDP container with a Signal 6 (SIGABRT).
+This behavior is caused by a CPU allocation setting from Cloud Run that causes background tasks processes (like the PDP), to de-allocate the CPU when not in use.
+This causes a termination of the container due to lack of CPU.
+
+To prevent this behavior, we added the `run.googleapis.com/cpu-throttling: false` annotation to the YAML.
+
+Read here for more information about CPU allocation settings in Cloud Run - https://cloud.google.com/run/docs/configuring/cpu-allocation
```

Comment:
A common misconfiguration issue that can be encountered when deploying the PDP to Google Cloud Run is  having the platform unexpectedly terminate the PDP container with a Signal 6 (SIGABRT). This issue stems from Cloud Run's CPU allocation settings, which allow background task processes, such as the PDP, to release CPU resources when idle. Consequently, this results in the container's termination due to insufficient CPU availability.
To address this issue and ensure uninterrupted service, we recommend adding the `run.googleapis.com/cpu-throttling: false` annotation in the YAML configuration.
For a deeper understanding of CPU allocation settings in Cloud Run and how they impact your applications, please refer to the following documentation: [Cloud Run CPU Allocation Settings](https://cloud.google.com/run/docs/configuring/cpu-allocation).